Source/WarpX.H

```diff
@@ -136,112 +138,160 @@ public:
     //
     // Functions used by implicit solvers
     //
-    void ImplicitPreRHSOp ( amrex::Real cur_time,
-                            amrex::Real a_full_dt,
-                            int         a_nl_iter,
-                            bool        a_from_jacobian );
+    void ImplicitPreRHSOp (amrex::Real cur_time, amrex::Real a_full_dt,
```

Comment:
Sometimes we use newlines in function declarations strategically to improve readability, e.g. :

```
void my_function (
    double Ex, double Ey, double Ez,
    double Bx, double By, double Bz,
    int i, int j, int k,
    bool flag);
```
I think that we may want to discuss if and how we would like `clang-format` to change this for us.

Reply:
I would agree here, that the layout of the arguments should be up to the developer to allow nice grouping.

Reply:
See https://github.com/BLAST-WarpX/warpx/pull/5739#discussion_r1990343527.

Reply:
Can the formatting of the routine arguments be turned off so that there is no standard? In this case, considering the wide variety of argument lists, there is no single standard that works well in all cases.